pkg/plugin/vm_restore_item_action.go

```diff
@@ -63,6 +65,11 @@ func (p *VMRestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (
 		return nil, errors.WithStack(err)
 	}
 
+	if runStrategy, ok := util.GetRestoreRunStrategy(input.Restore); ok {
+		p.log.Infof("Setting virtual machine run strategy to %s", runStrategy)
+		vm.Spec.RunStrategy = ptr.To(runStrategy)
```

Comment:
should make sure to unset `spec.running` when using runStrategy pretty sure webhook will deny otherwise

Reply:
ACK